docker-bake.hcl

```diff
@@ -80,6 +88,7 @@ target "op-node" {
     GIT_COMMIT = "${GIT_COMMIT}"
     GIT_DATE = "${GIT_DATE}"
     OP_NODE_VERSION = "${OP_NODE_VERSION}"
+    KONA_VERSION="${KONA_VERSION}"
```

Comment:
Do we really need to set KONA_VERSION when building op-node and all these other targets?